example/combined_reducers/index.dart

```diff
@@ -4,8 +4,8 @@ import 'dart:html';
 import 'package:redux/redux.dart';
 
 void render(AppState state) {
-  querySelector('#value').innerHtml = '${state.count}';
-  querySelector('#clickValue').innerHtml = '${state.clickCount}';
+  querySelector('#value')?.innerHtml = '${state.count}';
+  querySelector('#clickValue')?.innerHtml = '${state.clickCount}';
```

Comment:
I'd use `!.` here. It will throw if the `querySelector` returns `null`, just as the existing code.
The `?.` may hide errors if the elements get lost or renamed by accident.

Dittos below.